src/hu_stuff.c

```diff
@@ -1232,12 +1242,22 @@ static void HU_widget_build_coord (void)
 static void HU_widget_build_fps (void)
 {
   char hud_fpsstr[HU_MAXLINELENGTH/4];
-  extern int fps;
```

Comment:
Hm, do FPS still need their separate cheat code and widget then?

Reply:
Yes, I even think it should be in the QOL menu, not a cheat code. `SHOWFPS` is annoying to type.

Reply:
Okay.